modules/recommended-practices-for-node-hosts.adoc

```diff
@@ -0,0 +1,6 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id='recommended-practices-for-node-hosts-{context}']
+= Recommended practices for node hosts
```

Comment:
Are you planning to extend this file? If not, can you add the heading to the assembly?

Reply:
Whoops! This needs deleted out. Thought I already did that.